pkg/console/operator/sync_v400.go

```diff
@@ -125,40 +125,54 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	klog.V(4).Infof("sync loop 4.0.0 resources updated: %v", toUpdate)
 	klog.V(4).Infoln("-----------------------")
 
-	// the operand is in a transitional state if any of the above resources changed
-	// or if we have not settled on the desired number of replicas or deployment is not up to date.
-	if toUpdate {
-		co.HandleProgressing(updatedOperatorConfig, "ResourceSync", errors.New("Changes made during sync updates, additional sync expected."))
-	} else {
-		version := os.Getenv("RELEASE_VERSION")
+	co.HandleProgressing(updatedOperatorConfig, "ResourceSync", func() error {
```

Comment:
@jhadvig would def like a 2nd set of eyes on this.  Trying to break out these into individual units.

Note that `HandleAvailable()`, `HandleProgressing()`, `HandleDegraded()`, will prefix `Available`, `Progressing`, `Degraded`, in the given string `Reason`.  These will be aggregated by the `NewClusterOperatorStatusController()` to decide the final state.

We should be able to trust it instead of the nested `if` `else` `else if` we were using.